docs/src/forc/manifest_reference.md

```diff
@@ -99,3 +101,32 @@ Note that providing the corresponding cli options (like `--print-finalized-asm`)
 * print-intermediate-asm - false
 * print-ir - false
 * silent - false
+
+## The `[patch]` section
+
+The [patch] section of `Forc.toml` can be used to override dependencies with other copies. The example provided below patches <https://github.com/fuellabs/sway> source with master branch of the same repo.
+
+```toml
+[project]
+authors = ["user"]
+entry = "main.sw"
+organization = "Fuel_Labs"
+license = "Apache-2.0"
+name = "wallet_contract"
+implicit-std = false
+
+[dependencies]
+std = { git = "https://github.com/fuellabs/sway", branch = "master" }
```

Comment:
Does patching work with implicit `std` too?

Reply:
Yes, it works with implicit `std` too. To be more explicit and descriptive about what we are patching I created the doc like this. (i.e, explicitly added the dependency that I am patching). But I also realize that in this way the example is not in its most "default" form. 🤔

Reply:
I would prefer two subsections, one for patching a library that's not `std` or `core`, and one specifically for patching the implicit `std`.